### PR TITLE
Avatar badge added

### PR DESCRIPTION
**Pull request by t007rushi**

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -12,36 +13,35 @@
...
+  background-color: var(--avatar-prim-color);
+  border: 0.1rem solid var(--avat-sec-color);
+  width: 5rem;
 
-  height :5rem;
+  height: 5rem;
   border-radius: 50%;
   margin: 3rem;
   font-size: 3rem;
   text-align: center;
```
Comment (JantuDeb) [inline components/avatar.css]: You can remove `text-align: center`

File: components/avatar.css [hunk 2]
```diff
@@ -56,32 +56,30 @@
...
 
-.issue-border{
-    border : 0.2rem solid rgb(241, 12, 12)
+.issue-border {
+  border: 0.2rem solid var(--border-three);
 }
 
-@media screen and (max-width: 480px){
-.text_avatar {
-  display: flex;
```
Comment (JantuDeb) [inline components/avatar.css:76]: You can remove `display:flex` as it is the same for all screen sizes.

File: components/avatar.css [hunk 3]
```diff
@@ -56,32 +56,30 @@
...
-  flex-direction: column;
-  align-items: center;
-}
+@media screen and (max-width: 480px) {
+  .text_avatar {
+    display: flex;
```
Comment (JantuDeb) [inline components/avatar.css]: You can remove `display:flex` as it is the same for all screen sizes.

**Review by JantuDeb — APPROVED** [review]
I have mentioned some changes, you can include these changes and merge.

**Review by rushikesh3299 — COMMENTED** [review]
Jantu has given you some suggestions. Everything else looks good to me. You can merge this.

**Comment by t007rushi** [comment]
> I have mentioned some changes, you can include these changes and merge.

Thank You  Jantu

**Pull request merged**